scripts/db_migrator.py

```diff
@@ -181,9 +181,11 @@ def migrate_feature_table(self):
         '''
         feature_table = self.configDB.get_table('FEATURE')
         for feature, config in feature_table.items():
-            state = config.pop('status', 'disabled')
-            config['state'] = state
-            self.configDB.set_entry('FEATURE', feature, config)
+            state = config.get('status')
+            if state is not None:
+                config['state'] = state
+                config.pop('status')
+                self.configDB.set_entry('FEATURE', feature, config)
```

Comment:
The behavior is different, the removed code default status to 'disabled'. The added change didn't do that. Is this change intentional? If so, please clarify the intention.

Reply:
> The behavior is different, the removed code default status to 'disabled'. The added change didn't do that. Is this change intentional? If so, please clarify the intention.

Ying, The db_migrator should modify the field only if present. That's why the if check is added. If the field is not present in FEATURE_TABLE the db_migrator shouldn't assume it to be default disabled and leave it to the specific application which handles it to handle the default.